client/src/components/ElementEditor/Header.scss

```diff
@@ -0,0 +1,30 @@
+.element-editor-header {
+  display: flex;
+  justify-content: space-between;
+
+  .element-editor-header-title {
```

Comment:
See note in the component about the class names. The main benefit of using BEM class naming is that we don't need to nest the class definitions like this. If you agree and we update these then they can all be top level definitions with no nesting.

Reply:
I agree. Changes made, but not pushed yet.